object-manipulation-1/src/set-value.ts

```diff
@@ -0,0 +1,5 @@
+/* exported setValue */
+
+function setValue(obj: object, key: string, value: any): undefined {
```

Comment:
For functions that don't return value, use return type of "void" (instead of "undefined")